lib/interfaces/bdd.js

```diff
@@ -124,15 +124,15 @@ module.exports = function (suite) {
     };
 
     context.AfterSuite = function (fn) {
-      afterAllHooks.unshift(['AfterSuite', scenario.injected(fn, suites[0], 'afterSuite')]);
+      suites[0].afterAll('AfterSuite', scenario.injected(fn, suites[0], 'afterSuite'));
```

Comment:
this is revert changes of https://github.com/Codeception/CodeceptJS/commit/a79a7252b50b445f64fa2fd419badfc3a1cb7888#diff-70b2c1ced4ceeaeff1de3ac57c280b54

This was maked to set right ordering for after hooks.

Expected behavior should be : _beforeSuite hooks from helpers -> BeforeSuite from test -> _before hooks from helpers -> Before from test - > Test steps -> _failed hooks from helpers (if test failed) -> After from test -> _after hooks from helpers -> AfterSuite from test -> _afterSuite hook from helpers.

but if we will revert this changes then `_after` hooks from helpers will complete before `After` from test and `_afterSuite` hook from helpers will complete before `AfterSuite` from test.

Reply:
Ah ok. Thanks for the info. I’ll have another look at it then.